### ISSUE TITLE
Give choice to move blocked messages to Recycle Bin

### ISSUE THREAD

**Issue by hj-collab** [FossifyOrg/Messages#246]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Hi. I want the user be given the choice to send the blocked messages to the Recycle Bin. 

### Why do you want this feature?

That way they will know if any important message get blocked by the filters. I have the same setup for my mails. So, I check recycle bin once every 30 days to ensure that filters are working properly and none of my important mails are getting deleted.

**Comment by Aga-C** [comment]
It's a duplicate of https://github.com/FossifyOrg/Messages/issues/59.
